piccolo/apps/playground/commands/run.py

```diff
@@ -253,4 +256,14 @@ def run(
 
     from IPython.core.interactiveshell import _asyncio_runner  # type: ignore
 
-    IPython.embed(using=_asyncio_runner)
+    if ipython_profile:
+        print("Using user ipython profile")
+        print(
+            "More info: https://ipython.org/ipython-doc/3/config/intro.html#"
+        )
+        print("\n")
+        conf_args = {}
+    else:
+        conf_args = {"colors": "Neutral", "mouse_support": True}
```

Comment:
We might have to leave out `mouse_support`. When I try it on my Mac, it means I can no longer scroll.

Reply:
If I press the the option key, then I can scroll when using a trackpad.

Hmm, I think we should probably leave this out, as it could catch people out.

I presume someone can enable `mouse_support` in their iPython config if they want it.

Reply:
Yep, makes sense. I was sceptical of adding it myself